psl-core/src/main/java/org/linqs/psl/application/inference/InferenceApplication.java

```diff
@@ -109,14 +114,39 @@ protected void initialize() {
      */
     protected abstract void completeInitialize();
 
+    /**
+     * Minimizes the total weighted incompatibility of the GroundAtoms in the Database
```

Comment:
Just make this comment something short like:
"Alias for inference() with committing atoms."